itest/lnd_quiescence_test.go

```diff
@@ -0,0 +1,48 @@
+package itest
+
+import (
+	"github.com/btcsuite/btcd/btcutil"
+	"github.com/lightningnetwork/lnd/lnrpc"
+	"github.com/lightningnetwork/lnd/lnrpc/devrpc"
+	"github.com/lightningnetwork/lnd/lnrpc/routerrpc"
+	"github.com/lightningnetwork/lnd/lntest"
+	"github.com/stretchr/testify/require"
+)
+
+// testQuiescence tests whether we can come to agreement on quiescence of a
+// channel. We initiate quiescence via RPC and if it succeeds we verify that
+// the expected initiator is the resulting initiator.
+//
+// NOTE FOR REVIEW: this could be improved by blasting the channel with HTLC
+// traffic on both sides to increase the surface area of the change under test.
+func testQuiescence(ht *lntest.HarnessTest) {
+	alice, bob := ht.Alice, ht.Bob
+
+	chanPoint := ht.OpenChannel(bob, alice, lntest.OpenChannelParams{
+		Amt: btcutil.Amount(1000000),
+	})
+	defer ht.CloseChannel(bob, chanPoint)
+
+	res := alice.RPC.Quiesce(&devrpc.QuiescenceRequest{
+		ChanId: chanPoint,
+	})
+
+	require.True(ht, res.Initiator)
```

Comment:
What about initiation by the non-initiator? Or when both of them try to initiate at the same time?

Reply:
> Do we have a means of blocking transmission in the test suite?

We usually achieve this by shutting down the node.

Reply:
Unfortunately that won't work here because shutting down the node would result in disconnection and that would reset the quiescence state.

Reply:
Yeah right - then we can also wait somewhere like when receiving the msg. There's the `dev` config we can use here, for reference, this is how we test a behavior by sleeping only in our itest,
https://github.com/lightningnetwork/lnd/blob/f34bf51d164fe0472f1c0e354ae38f20ba9fd490/funding/manager.go#L3888-L3891

Reply:
I'm investigating that now.

Reply:
Ok I definitely do not want to add this type of branching into the link code. I don't think the clutter is worth it.